ghost/i18n/locales/sk/portal.json

```diff
@@ -13,11 +13,11 @@
     "+1 (123) 456-7890": "",
     "A login link has been sent to your inbox. If it doesn't arrive in 3 minutes, be sure to check your spam folder.": "Prihlasovací odkaz bol odoslaný na váš e-mail. Ak nedorazi do 3 minút, skontrolujte priečinok so spamom",
     "Account": "Účet",
-    "Account details updated successfully": "",
+    "Account details updated successfully": "Podrobnosti o účte boli úspešne aktualizované",
     "Account settings": "Nastavenia účtu",
     "After a free trial ends, you will be charged the regular price for the tier you've chosen. You can always cancel before then.": "Po uplynutí skúšobného obdobia vám bute účtovaná bežná cena pre vybrunú úrovenň. Vždy to môžte pred tým zrušiť",
```

Comment:
This is not your change, but would you mind checking the grammar and spelling of this string (line 18)?  AI is flagging it as having spelling and grammar errors.  Thanks!